tests/test_tally.py

```diff
@@ -436,6 +436,63 @@ def test_mcnp6_tally_vol_proton_volume_set():
                  "F14:h 12\n"+\
                  "SD14 100.000000\n",tally.mcnp(1,"mcnp6"))
 
+def test_mcnp_mesh_tally_xyz():
+    particle = "Neutron"
+    geometry = "Cartesian"
+    origin = [1,2,3]
+    i = [5, 10, 20, 25]
```

Comment:
Suggests to use `i_mesh`, `j_mesh` and `k_mesh` to replace `i, j, k` here for better understanding.